app/metrics.yaml

```diff
@@ -468,6 +468,23 @@ events:
     metadata:
       tags:
         - Sharing
+  save_to_pdf_failure:
+    type: event
+    description: |
+      A user tapped the save pdf but an error ocurred
+      and the process failed.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/27635
+    data_reviews:
+      - ?????
```

Comment:
Please don't forget to update this URL.  Thanks

Reply:
It should be https://github.com/mozilla-mobile/fenix/pull/27661#issuecomment-1300505370, which points to review approval.